tests/test092.cpp

```diff
@@ -11,7 +11,7 @@ int main()
   std::string loc = "de_DE"; // uses comma (,) as decimal separator
   try
   {
-    std::locale::global(std::locale(loc));
+    std::locale::global(std::locale(loc.c_str()));
```

Comment:
Second build error with GCC 4.8.5, that appears after modifying `src/rapidcsv.h`. Error is:

```
$ ./make.sh build
...
/data_local/l0440448/rapidcsv/tests/test092.cpp: In function ‘int main()’:
/data_local/l0440448/rapidcsv/tests/test092.cpp:14:40: error: no matching function for call to ‘std::locale::locale(std::string&)’
     std::locale::global(std::locale(loc));
                                        ^
```
[(full log 2nd error)](https://github.com/d99kris/rapidcsv/files/12016158/make-build-gcc-4.8.5.error-2.txt)